sale_loyalty_general_discount_promo_code/tests/test_sale_loyalty_general_discount_promo_code.py

```diff
@@ -0,0 +1,167 @@
+from odoo import fields
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
```

Comment:
```suggestion
from odoo.addons.base.tests.common import BaseCommon
```

please use it instead